### PR TITLE
Fusebox update and social buttons

### PR DESCRIPTION
**Pull request by l2succes**

### CODE REVIEW

File: src/components/buttons/facebook.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import * as React from 'react'
+import Button, { ButtonProps } from './default'
+import Icon, { IconProps } from '../icon'
```
Comment (alloy) [inline src/components/buttons/facebook.tsx]: IconProps is unused

File: src/components/buttons/twitter.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import * as React from 'react'
+import Button, { ButtonProps } from './default'
+import Icon, { IconProps } from '../icon'
```
Comment (alloy) [inline src/components/buttons/twitter.tsx]: Same

File: src/components/buttons/facebook.tsx (hunk 1 above)
Comment (l2succes) [inline src/components/buttons/facebook.tsx]: Good catch

**Comment by alloy** [comment]
Dope, very fast indeed

**Pull request merged**